cli/js/ops/net.ts

```diff
@@ -17,7 +18,7 @@ export type Addr = NetAddr | UnixAddr;
 export enum ShutdownMode {
   // See http://man7.org/linux/man-pages/man2/shutdown.2.html
   // Corresponding to SHUT_RD, SHUT_WR, SHUT_RDWR
-  Read = 0,
+  Read,
   Write,
```

Comment:
This should be more explicit, not less explicit. Please change to:
```
Read = 0,
Write = 1,
```
These cannot be any other value.
https://github.com/denoland/deno/blob/5c43131be1a36671fd564b82709e11dff66488f3/cli/ops/net.rs#L366-L370 When code explicitly gives values to enums, it's probably not a mistake. It's a hint to the programmer that they should not be ordered differently.